projects/github-actions/required-review/src/request-review.js

```diff
@@ -0,0 +1,46 @@
+const core = require( '@actions/core' );
+const github = require( '@actions/github' );
+const getUsername = require( './get-username.js' );
+
+/**
+ * Request review from the given team
+ *
+ * @param {string} team - GitHub team slug, or @ followed by a GitHub user name.
+ */
+async function requestReviewer( team ) {
+	const octokit = github.getOctokit( core.getInput( 'token', { required: true } ) );
+	const owner = github.context.payload.repository.owner.login;
+	const repo = github.context.payload.repository.name;
+	const pr = github.context.payload.pull_request.number;
+
+	if ( team.startsWith( '@' ) ) {
+		// Handle @singleuser virtual teams. Fetch the correct username case from GitHub
+		// to avoid having to worry about edge cases and Unicode versions and such.
```

Comment:
Is that a concern here? We do it in the other place because we're comparing the name from the config with the name returned on the review. But here I'd hope submitting "someusername" instead of "someUserName" would just do the right thing.

Reply:
Tested, and yes `someUserName` works just as well as `someusername`. So guess there's really no reason to factor out [this code](https://github.com/Automattic/jetpack/pull/30653/files#diff-a2d2e2075dd774cef44c3db803766dcf96aeb8486b1580b61deb9ac28c144674L26) as here we can just use `const login = team.slice( 1 );` and not touch `team-members.js`.

If you agree, I'll just put that code back so I'm touching fewer files.